src/metrics/test/metrics_test.cpp

```diff
@@ -242,6 +242,33 @@ TEST_CASE("metrics output format") {
         reporter.report<ReporterClockSourceStub>(out, "csv");
         REQUIRE(out.str() == expected);
     }
+
+    SECTION("cedar-csv reporting") {
+        auto expected =
```

Comment:
I'm going to put this string into its own file in #175. Feel free to use it for this test as well. Sharing a file could be useful for picking up format changes on the Python side without having to run genny live.

Reply:
I've tentatively placed the file in `src/python/tests/fixtures/cedar/shared_with_cxx_metrics_test.csv`

See: https://github.com/mongodb/genny/pull/175/commits/f85df473566f438a73d3ac6d921b58b17ab72c88

Reply:
I attempted to change metrics_test.cpp to read the `src/python/tests/fixtures/cedar/shared_with_cxx_metrics_test.csv` file in 359743a9c1d8862feae90e6b82e82d254195c6ae but ultimately back those changes out c99d12655d2b4ff68d6662bbd4b3b3ab78bdc7da in when I realized that my preferred way of running `./build/src/metrics/metrics_test` isn't compatible without we're running `ctest` out of the `./build/src/metrics` directory.

Reply:
Ah oh well. Thanks for trying it out!

Reply:
After reading some creative solutions to reading files in C++ on SO, I'm convinced it's far easier to just have python parse the C++ source file and extract this string.

Reply:
Sorry for the vagueness of the above comment; I posted it mostly for my own amusement.

A more elaborate description of a mechanism for C++ and Python tests to share fixtures is here: https://jira.mongodb.org/browse/TIG-1479

Reply:
(we could easily "findup" the repo root by looking for `.git` and agree on the path from there. but fine to have this convo in TIG-1479)